.github/workflows/claude.yaml

```diff
@@ -0,0 +1,66 @@
+name: Claude
+on:
+  issues:
+    types:
+      - opened
+      - assigned
+  issue_comment:
+    types:
+      - created
+  pull_request:
+    paths:
+      - .github/workflows/claude.yaml
+  pull_request_review:
+    types:
+      - submitted
+  pull_request_review_comment:
+    types:
+      - created
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.issue.number || github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: false
+jobs:
+  default:
+    if: |
+      (github.event_name == 'issue_comment' && contains(github.event.comment.body, '@claude')) ||
+      (github.event_name == 'pull_request_review_comment' && contains(github.event.comment.body, '@claude')) ||
+      (github.event_name == 'issues' && contains(github.event.issue.body, '@claude')) ||
+      (github.event_name == 'pull_request_review' && contains(github.event.review.body, '@claude')) ||
+      github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
Watching this go on Connect and occasionally getting stuck, I think this timeout could be lower--if it succeeds, it only takes a few minutes.